.travis.yml

```diff
@@ -22,7 +32,23 @@ install:
 script:
   - make
   - make travis-presubmit
-  - ./coverage.sh 
+  - ./coverage.sh
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+
+deploy:
+  provider: script
+  script:
+    - sudo apt-get update
+    - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-engine
+    - pip install docker-compose
+    - docker --version
+    - docker-compose --version
+    - if [ ! -d "$HOME/google-cloud-sdk/bin" ]; then rm -rf $HOME/google-cloud-sdk;
+      curl https://sdk.cloud.google.com | bash; fi
+    - openssl aes-256-cbc -K $encrypted_a1eb99cfc21e_key -iv $encrypted_a1eb99cfc21e_iv -in service_key.tar.gz.enc -out service_key.tar.gz -d
```

Comment:
Double check that `service_key.tar.gz.enc` is still correct.  It doesn't look like it matches with the `.enc` file in the PR.  

Consider naming this `travis_secrets.tar.gz.enc`